.NET/Microsoft.Recognizers.Text.Sequence/Config/PhoneNumberConfiguration.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Microsoft.Recognizers.Text.Sequence
+{
+    public class PhoneNumberConfiguration : ISequenceConfiguration
+    {
+        public PhoneNumberConfiguration(SequenceOptions options = SequenceOptions.None)
+        {
+            Options = options;
+        }
+
+        public SequenceOptions Options { get; }
+
+        public Regex BRPhoneNumberRegex { get; set; }
```

Comment:
This seems too repetitive. Ideally the config should have just the exact piece that changes.